lib/Web/Machine/Resource.pm

```diff
@@ -57,6 +58,17 @@ sub base_uri                  { undef }
 sub process_post              { 0 }
 sub content_types_provided    { [] }
 sub content_types_accepted    { [] }
+
+sub get_acceptable_content_type_handler {
+    my $self = shift;
+    Web::Machine::FSM::States::_get_acceptable_content_type_handler( $self, $self->request );
```

Comment:
I don't really like having this code call a private function in another package. Maybe this belongs in Util?